spec/factories/factories.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/pivotal_tracker'
+
+FactoryGirl.define do
+  skip_create
+  
+  factory :me, class: PivotalTracker::Me do
+    api_token '0'
+    created_at DateTime.now
+    email 'example@example.com'
+    has_google_identity true
+    id 1
```

Comment:
@jasonnoble any tips on how to implement? My google search is suggesting a random number from a range. Should this be a sequence?